packages/core/src/electron-browser/window/electron-window-preferences.ts

```diff
@@ -0,0 +1,61 @@
+/********************************************************************************
+ * Copyright (C) 2021 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { interfaces } from 'inversify';
+import { createPreferenceProxy, PreferenceContribution, PreferenceProxy, PreferenceSchema, PreferenceService } from '../../browser/preferences';
+
+export namespace ZoomLevelConstants {
+    export const ZOOM_DEFAULT = 0;
+    // copied from https://github.com/microsoft/vscode/blob/dda96b69bfc63f309e60cfc5f98cb863c46b32ac/src/vs/workbench/electron-sandbox/actions/windowActions.ts#L47-L48
+    export const ZOOM_MIN = -8;
+    export const ZOOM_MAX = 9;
+    // amount to increment or decrement the window zoom level.
+    export const ZOOM_VARIATION = 0.5;
+}
+
+export const windowPreferencesSchema: PreferenceSchema = {
```

Comment:
It will help when we then have the generic `windowPreferences`.

```suggestion
export const electronWindowPreferencesSchema: PreferenceSchema = {
```